README.md

```diff
@@ -6,6 +6,7 @@
 - HTML
 - CSS
 - JAVASCRIPT
+- lUXON
 - VSCODE
 
 ## Live Demo
```

Comment:
- [ ] After the project's name(**Awesome-Books**), kindly provide a detailed description of a project you have worked on. 


- [ ] Also, kindly provide detailed instructions on how to set up the project locally.` Download the folder and click on index.html` is not enough for someone who doesn't have enough skill for coding to set up the project locally. 
- [ ] Kindly edit your **Acknowledgment** section. Always try to double-check your README file even after doing copy and pasting.
- [ ] Lastly, fix your License link. It's broken. This will render your README file professional.